pkg/transforms/common.go

```diff
@@ -156,6 +156,53 @@ func CommonEdges(uid string, ns NodeStore) []Edge {
 
 	// deployer subscriber edges
 	ret = append(ret, edgesByDeployerSubscriber(nodeInfo, ns)...)
+
+	ret = edgesByKyverno(ret, currNode, ns)
+
+	return ret
+}
+
+// Function to create an edge linking any resource with a Kyverno Policy or ClusterPolicy that generates the resource.
+func edgesByKyverno(ret []Edge, currNode Node, ns NodeStore) []Edge {
+	labels, ok := currNode.Properties["label"].(map[string]string)
```

Comment:
This could throw a runtime error if `currNode.Properties["label"]` is nil. I think of a specific scenario where that would happen, but let's add a safety check just in case.

Reply:
Thanks for confirming! 

FYI. Interestingly, if I write it differently I get a runtime panic.

```go
props := make(map[string]interface{}, 0)
l, ok := props["label"]
label := l.(map[string]string)
```
```
panic: interface conversion: interface {} is nil, not map[string]string
```

Reply:
`label, ok := l.(map[string]string)`
maybe work?!

Reply:
Your code works well, no changes needed here.

My last comment was just FYI to give more context about why I asked to double check this code.